users/users.go

```diff
@@ -66,14 +71,23 @@ func isEmail(email string) bool {
 		return false
 	}
 
-	user := email[:at]
+	local := email[:at]
 	host := email[at+1:]
 
-	if len(user) > 64 {
+	if len(local) > maxLocalLen || len(host) > maxHostLen {
```

Comment:
You could then check here `if len(local) == 0 || ...`

Reply:
ok